src/main/java/hudson/plugins/jira/JiraCreateIssueNotifier.java

```diff
@@ -39,7 +39,7 @@
 import org.kohsuke.stapler.AncestorInPath;
 import org.kohsuke.stapler.DataBoundConstructor;
 import org.kohsuke.stapler.QueryParameter;
-import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerRequest2;
```

Comment:
I thought the times of having `2` at the end of things is long gone 😆 
How about moving this class to `org.kohsuke.stapler.ee9.StaplerRequest`?

Reply:
This shipped in Jenkins core months ago and is now part of the public API which cannot be changed.